python/ray/tune/schedulers/pbt.py

```diff
@@ -228,6 +234,7 @@ def _log_config_on_step(self, trial_state, new_state, trial,
 
         For each step, logs: [target trial tag, clone trial tag, target trial
         iteration, clone trial iteration, old config, new config].
+
```

Comment:
```suggestion
```

Reply:
hmm it says your comment was marked as resolved. do you want me to remove the empty lines at the end of docstrings again?

Reply:
Ah I checked the style guide and what you have is right.